src/store-update.android.ts

```diff
@@ -13,19 +13,30 @@ app.on(app.resumeEvent, () => {
   StoreUpdate.checkForUpdate()
 })
 
-export class StoreUpdate extends StoreUpdateCommon {
-  private static _checkHasTimeouted = false
-  private static _timeoutChecker
-  private static _looperChecker
+export class StoreUpdate {
+  private static _storeUpdateCommon
```

Comment:
I would name it `_common` instead of `_storeUpdateCommon`, as it's a bit redondant

`StoreUpdate._storeUpdateCommon` -> `StoreUpdate._common`